.github/workflows/build.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ ubuntu-latest ]
-        java: [ 8, 11, 15, 17, 18, 19 ]
```

Comment:
It was not really beneficial, as we already use version 8 when building. 

To onboard the spotless update we would need to make our pipeline even more complicated/conditional based on the java version used but I don't see any real reason to do so.